pages/linux/nautilus.md

```diff
@@ -0,0 +1,17 @@
+# nautilus
+
+> Default file explorer for GNOME desktop environment.
+> Also known as GNOME Files.
+> More information: <https://manned.org/nautilus>.
+
+- Launch Nautilus:
+
+`nautilus`
+
+- Launch Nautilus as root user:
+
+`sudo nautilus`
+
+- Launch Nautilus and display a given directory:
+
+`nautilus {{path/to/file_or_directory}}`
```

Comment:
Actually I'd split those two in two different examples. Because when you specify a directory, nautilus opens at that location, but when you specify a file, nautilus opens the parent folder and just highlights the file.